.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,98 @@
+            - name: Checkout
+              uses: actions/checkout@v2.3.1 # If you're using actions/checkout@v2 you must set persist-credentials to false in most cases for the deployment to work correctly.
+              with:
+                persist-credentials: false
+            #
+            # Generate documentation
+            #
+            - name: Install eve and dependencies
+              run: |
+                  python3 -m pip install .
+                  python3 -m pip install -r requirements_dev.txt # TODO remove (once we don't include debugtools)
+            - name: Build eve documentation
+              run: |
+                cd docs/eve
+                make html
+            - name: Build gtc documentation
+              run: |
+                cd docs/gtc
+                make html
+            - name: Build gtc_frontend documentation
+              run: |
+                cd docs/gt_frontend
+                make html
+            #
+            # Deploy documentation
+            #
+            - name: Install dependencies # rsync -> github-pages-deploy-action, jq -> PR id
+              run: apt-get update && apt-get install -y jq rsync
+            - name: Gather html documentation
+              shell: bash
+              run: |
+                mkdir _deploy/
+                cd _deploy/
+                mv ../docs/eve/_build/html/ eve
+                mv ../docs/gtc/_build/html/ gtc
+                mv ../docs/gt_frontend/_build/html/ gt_frontend
+            - name: Capture current branch name
+              shell: bash
+              id: current_branch
+              run: echo "::set-output name=name::$(git branch --show-current)"
+            - name: Check if current commit is the most recent one on this branch
+              shell: bash
+              id: check_if_most_recent_upstream
+              run: |
+                REMOTE_URL=$(git remote get-url origin)
+                LATEST_UPSTREAM_COMMIT=$(git ls-remote $REMOTE_URL | grep $GITHUB_REF | cut -f 1)
+                CURRENT_COMMIT=$(git rev-parse HEAD)
+                if [ "$LATEST_UPSTREAM_COMMIT" = "$CURRENT_COMMIT" ]; then echo "::set-output name=is_most_recent::true"; fi
+            - name: Print warning if deployment skipped
+              run: echo "::warning::Documentation deployment skipped since the current commit is not the most recent of this branch"
+              if: steps.check_if_most_recent_upstream.outputs.is_most_recent != 'true'
+            - name: Assemble target folder
+              id: target_folder
+              shell: bash
+              run: |
+                if [ "${{ github.event_name }}" = "pull_request" ]; then
+                  PR_ID=$(jq --raw-output .pull_request.number "$GITHUB_EVENT_PATH")
+                  if ! [[ "$PR_ID" =~ ^[0-9]+$ ]]; then exit 1; fi
+                  TARGET_FOLDER="docs/pr/${PR_ID}"
+                else
+                  TARGET_FOLDER="docs/branches/$(git branch --show-current)"
```

Comment:
Since it's now building only master shouldn't the `TARGET_FOLDER` always be `latest` or something like this?

Reply:
Done